public/index.html

```diff
@@ -7,11 +7,22 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <h1>To-Do List:</h1>
+    <h1>To-Do List</h1>
     <div id="app">
-      <input type="text" id="new-task" placeholder="Add a new task" />
-      <button id="add-task">Add Task</button>
-      <ul id="task-list"></ul>
+      <div class="input-container">
+        <input type="text" id="new-task" placeholder="Add a new task" />
+        <button id="add-task">Add Task</button>
+      </div>
+      <div class="cards-container">
```

Comment:
Task Containers Structure:
Issue: The structure here looks fine, but it would be better to assign unique IDs for each list (todo-list and completed-list) if you plan to differentiate between them in JavaScript. The current structure suggests you're using the same class (task-list) for both, which is fine if you're applying shared styles but might be confusing for JavaScript interactions.

Recommendation: Keep the classes for shared styles, but make sure you're differentiating the two lists by ID for JavaScript logic.

Reply:
Hi @KhanyiCodes, Changes have been made accordingly. 

Thanks for reviewing my code :)